cypress/integration/1216-menu/345-user_sees_a_menu_when_logged_in.feature

```diff
@@ -6,5 +6,5 @@ Feature: User sees a menu when logged in
         And they have no NVA role
         When they look at any page in NVA
         Then they see a menu containing
-            | My profile |
-            | Log out    |
+            | My Profile |
+            | Log Out    |

```

Comment:
hvorfor veksler disse mellom stor og liten forbokstav hele tiden?

Reply:
Scenarioene har ikke vært oppdatert og hver gang jeg kjører `pull-features` så blir de endret til det som ligger i NVA-scenarios. Sjekker om det faktisk er korrekt i NVA-scenarios nå.

Reply:
PR'en jeg har inne for NVA-scenarios fikser dette.